src/main/resources/GoogleApps/GoogleAppsConfigClass.xml

```diff
@@ -305,82 +305,85 @@
     </property>
     <property>
       <code>require(['jquery'], function (jQuery) {
-    var p = "#GoogleApps\\.GoogleAppsConfigClass_0_";
 
-    //deactivating
-    function deactivate(elts) {
+        var prefix = '#GoogleApps\\.GoogleAppsConfigClass_0_';
+
+        //deactivating
+        function deactivate(elts) {
         elts.each(function () {
-            jQuery(jQuery(this).closest("dl")).find("label").css("color","darkgrey");
-            this.wasDisabled = jQuery(this).prop("disabled");
-            jQuery(this).prop("disabled", true);
+        jQuery(jQuery(this).closest('dl')).find('label').css('color','darkgrey');
+        this.wasDisabled = jQuery(this).prop('disabled');
+        jQuery(this).prop('disabled', true);
         });
-    }
+        }
 
 
-    // reactivating
-    function reactivate(elts) {
+        // reactivating
+        function reactivate(elts) {
         jQuery(elts).each(function () {
-            if (typeof (this.wasDisabled)) {
-                jQuery(jQuery(this).closest("dl")).find("label").css("color","black");
-                jQuery(this).prop("disabled", this.wasDisabled);
-            }
+        if (typeof (this.wasDisabled)) {
+        jQuery(jQuery(this).closest('dl')).find('label').css('color','black');
```

Comment:
In order to have a nice display on any color theme, it is recommended to use some variables defined in https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/colorThemeInit.vm line `$theme.textColor` instead of `black` and `darkgrey`. But this can be fixed in a future issue.

Reply:
Very good point!
I couldn't find a color for "deactivated" text which shows inside the text that there's little to search or read. I could include a script to mix the the textColor and the background colour at about 80% (as [one here](https://stackoverflow.com/questions/14482226/how-can-i-get-the-color-halfway-between-two-colors#14482509)) but that seems a bit too big. Do you know if there's a deactivated-text colour?